com.bkmarriott.reservation/src/main/java/com/bkmarriott/reservationservice/reservation/presentation/rest/controller/InventoryCommandController.java

```diff
@@ -0,0 +1,33 @@
+package com.bkmarriott.reservationservice.reservation.presentation.rest.controller;
+
+import com.bkmarriott.reservationservice.reservation.application.service.InventoryService;
+import com.bkmarriott.reservationservice.reservation.presentation.rest.dto.command.InventoryModification.Response;
+import com.bkmarriott.reservationservice.reservation.presentation.rest.util.reponse.ApiResponse;
+import com.bkmarriott.reservationservice.reservation.presentation.rest.util.reponse.ApiResponse.Success;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@Slf4j
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/v1/reservation/inventory")
```

Comment:
```suggestion
@RequestMapping("/api/v1/reservations/inventories")
```
RESTful API는 보통 URL을 복수형으로 표기합니다!